contrib/ucx_perftest_config/test_types_ucp_daemon

```diff
@@ -0,0 +1,4 @@
+#
+# UCP daemon basic
+#
+ucp_daemon_bw                        -t ucp_am_bw -w 0 -n 10000 -s 8192
```

Comment:
why can't we reuse test_types_ucp? or just run ucx_perftest without batch file (since in this batch file there is only one test)

Reply:
For now we cannot reuse `test_types_ucp` because of multiple limitations:
- tag matching is not supported
- only contig buffers are supported
- only single thread is supported
- latency tests need to be adjusted to daemon workflow
If/when these limitations are addressed, then we can reuse this test suite.

Regarding batch vs single test, I'm absolutely sure that we need a batch mode for the future improvements (out-of-the-box XGVMI task). For instance, we need to add a test for short messages once we force XGVMI to always use RNDV. I also plan to add am_lat tests to this batch in the future. So IMO having a separate test batch is the right choice

Reply:
ok